scripts/deploy-contracts.js

```diff
@@ -317,6 +317,17 @@ async function main() {
     console.log("\tMinting 1 mil tokens to owner");
     ownershiptx = await ocean.connect(owner).mint(owner.address,ethers.utils.parseUnits('1000000', 'ether'), options);
     await ownershiptx.wait()
+    //mint tokens in barge
+    if (process.env.MNEMONIC && networkDetails.chainId==8996){
```

Comment:
I see that there isn't an envvar to check whether to do this.

How long does it take to mint the OCEAN? If it's too long, we should seriously consider making this optional.